### DISCUSSION TITLE
Print tables diagram in Tabular Editor 3

### DISCUSSION THREAD

**Discussion by pgschr** [TabularEditor/TabularEditor3#1352] [Data Modeling]
Is there a way to print the tables diagram in Tabular Editor 3? or export to PDF?

**Comment by mlonsk** [comment]
Hi @pgschr 

You can save the diagrams as a .te3diag file and reopen them later inside Tabular Editor 3, but you cannot currently print or export them. 
I would suggest taking a screenshot if you need to export them.

Can you tell me a bit further about your use case for printing/exporting so we can see how we best can implement such a feature in the future.

**Comment by msloane794pro** [comment]
I also have a need to print/export diagrams from Tabular Editor 3.
Here are some use-cases that I would like to see satisfied:
1. The ability to incorporate diagrams from Tabular Editor 3 into documents (such as MS Word, MS PowerPoint, Google Docs).
2. The ability to view diagrams created from Tabular Editor 3 without installing the Tabular Editor 3 application.

If Tabular Editor had the capability to export diagrams in PDF or some standard image format, my needs would be met.

**Reply by mlonsk** [reply]
Hi @msloane794pro 

Thank you for the additional context; much appreciated.

As I mentioned in my previous column, the current best workaround is to take a Screenshot of the diagram and save it in the format or document you need.

We have the idea on our backlog, but it is not part of our immediate roadmap.
